src/main/java/mServer/crawler/sender/MediathekDw.java

```diff
@@ -220,7 +221,13 @@ private void laden2(String urlThema, String thema, String titel, String urlSendu
                 addFilm(film);
             }
 
-        }
-
+        }        
     }
+    
+    private static String addUrlPrefixIfNecessary(String url) {
+        if (!url.startsWith(HTTP)) {
+            url = ADDURL + url;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "url". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)